scripts/lib/mkcloud-common.sh

```diff
@@ -539,3 +539,42 @@ function dist_to_image_name
 }
 
 # ---- END: functions related to repos and distribution settings
+
+function qacrowbarsetup_help
```

Comment:
If this is help specific to `qacrowbarsetup.sh` then why is it in a file called `mkcloud-common.sh`? Wouldn't `qacrowbarsetup-help.sh` or `qacrowbarsetup-shared.sh` make more sense?